seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/FlinkEnvironment.java

```diff
@@ -322,4 +326,61 @@ private void setCheckpoint() {
         }
     }
 
+    public StreamExecutionEnvironment creatMetricStreamEEnvironment() {
+
+        if(!config.hasPath(ConfigKeyName.Metric_Class)){
+            return StreamExecutionEnvironment.getExecutionEnvironment();
+        }
+        //构建flink-metrics参数
+        ConfigOption<String> REPORTERS_LIST =
```

Comment:
Maybe you can create an new method called `initMetricConfig(Configuration config)` to cover this logic.